index.js

```diff
@@ -99,10 +122,6 @@ class PhotoViewManager {
       if (this.scale === 1) {
         return;
       }
-
-      e.srcEvent.stopPropagation();
```

Comment:
I am not sure if it's good to remove this. We can actually stop the propagation of `PointerEvents` also